### ISSUE TITLE
Please update npm package !

### ISSUE THREAD

**Issue by catalinmiron** [lelandrichardson/react-native-parallax-view#9]
Thanks

**Comment by hokster7** [comment]
I don't know if what I've run into is related:

I ran `npm i react-native-parallax-view --save` and it ran into this error:

``` node
npm WARN peerDependencies The peer dependency react-native@^0.4.1 included from react-native-parallax-view will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.

> ws@0.4.31 install /Users/fushunpoon/Development/projects/makeeatsee/ActiveExperiments/ReactNativePreviewCard/node_modules/react-native-parallax-view/node_modules/react-native-blur/node_modules/react-native/node_modules/ws
> (node-gyp rebuild 2> builderror.log) || (exit 0)

  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/bufferutil.node
  SOLINK_MODULE(target) Release/bufferutil.node: Finished
  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/validation.node
  SOLINK_MODULE(target) Release/validation.node: Finished
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "i" "react-native-parallax-view" "--save"
npm ERR! node v0.10.35
npm ERR! npm  v2.7.6
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react-native does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-native-parallax-view@1.0.2 wants react-native@^0.4.1

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/fushunpoon/Development/projects/makeeatsee/ActiveExperiments/ReactNativePreviewCard/npm-debug.log
```

**Comment by lelandrichardson** [comment]
I've updated the repo to use the new Animated API. The big catch is that it requires >=0.8.0, and as a result, I've upped the npm version number to 2.0.

Because it requires 0.8.0, this may yet again throw peer dependencies (but for the opposite reason) depending on what version of RN you are using.

I've tried to have the version number as flexible as possible by including the rc packages as well as the straight version numbers.

Nevertheless, I will be paying more attention to this repo moving forward. sorry about that!
